src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -414,23 +402,18 @@ class MyCheck extends AbstractJavadocCheck {
       </p>
 
       <p>
-        To make Checkstyle support HTML5 tags whose end tag is optional (omittable) and HTML5 void
-        elements we should update Javadoc Parser because each element that breaks
-        <a href="#Tight-HTML_rules">Tight-HTML rules</a> have to be defined in Javadoc grammar.
-        In the future, we should update Javadoc grammar if those tag lists extend (new tags,
-        new HTML standard, etc.).
-        (We already have an <a href='https://github.com/checkstyle/checkstyle/issues/3332'>issue
-        on updating Javadoc grammar to HTML5</a>)
+        Checkstyle does not support HTML5 tags with optional end tags. See
+        <a href='https://github.com/checkstyle/checkstyle/issues/3332'/> for details.
```

Comment:
1) Switch to double quotes
2) Why is this self closing? This means this link has no clickable text.